types/storybook__react/index.d.ts

```diff
@@ -16,7 +17,7 @@ export type StoryDecorator = (story: RenderFunction, context: { kind: string, st
 
 export interface Story {
     readonly kind: string;
-    add(storyName: string, callback: RenderFunction): this;
+    add(storyName: string, callback: RenderFunction, parameters?: { [key: string]: any }): this;
```

Comment:
afaik there are also global parameters that can be set - they are merged together with the local ones. Do you want to take a look and add these if they are exposed via API? Would be good to get all parameters into the types in one go, if possible.

Reply:
I've added the other pieces of the API related to parameters. Thanks for the suggestion.